BearlyHumans/squirrelville
# Improved HUD

This PR greatly improves the HUD based on Luke's design.
https://github.com/BearlyHumans/squirrelville/wiki/In-Game-HUD

Steps to test objectives sign:
1. Be on Jordan scene
2. Interact with the tall capsule NPC to receive an objective
3. Pause the game to check if the objective is also displayed

Steps to test food counter:
1. Be on Jordan scene
2. Eat food
3. Attempt to turn into ball form with different amounts of food to see if it matches position of circle

https://user-images.githubusercontent.com/62001991/132464328-63dc46c2-4955-4740-885a-12e6cbc8b771.mp4
